### ISSUE TITLE
Daily submission limit

### ISSUE THREAD

**Issue by sachaduvivier** [allegro/allegro-api#10532]
Dec 18, 2024 at 12:34:14.179 am

Each day, we're doing 600 calls to allegro api at the same time, but we're always getting User exceeded suggestions limit, so we can't update ours products 

Example:
HTTP/2 429  returned for "https://api.allegro.pl/sale/products/f6b27f4d-cc10-44a7-a25f-228668c925dc/change-proposals".
HTTP/2 429  returned for "https://api.allegro.pl/sale/products/01a45017-e960-48c3-bd96-5485cdd84233/change-proposals".

{"errors":[{"code":"UserExceededSuggestionsLimit","message":"User exceeded suggestions limit (user = 12072XXXX)","details":null,"path":"/sale/products/f6b27f4d-cc10-44a7-a25f-228668c925dc/change-proposals","userMessage":"You have exceeded your daily submission limit. Please try again tomorrow.","metadata":{}}]}

**Comment by AureliuszBrussy** [comment]
Yes, the resource is limited and a single user can send a limited number of suggestions per day. I'm sorry, but I can't do anything about it.

**Comment by sachaduvivier** [comment]
I understand, but how much requests can we do per day then?

**Comment by AureliuszBrussy** [comment]
This resource is limited to 100 suggestions per day for a single user.
